main.nf

```diff
@@ -3,50 +3,56 @@
 
 // check parameters
 if (!params.inputDir) {
-    exit 1, "inputDir parameter is missing."
-}
-if (!params.outputDir) {
-    exit 1, "outputDir parameter is missing."
+    exit 1, 'inputDir parameter is missing.'
 }
 if ( !new File(params.inputDir).exists()) {
     exit 1, "The input folder does not exists."+params.inputDir+"\n"
 }
-if (!params.outPrefix) {
-    exit 1, "Output prefix parameter is missing."
+if (!params.outputDir) {
+    exit 1, "outputDir parameter is missing."
 }
-if (!params.debug) {
-    exit 1, "Debug prefix parameter is missing."
+if (!params.starTmpDir) {
+    exit 1, "starTmpDir parameter is missing."
+}
+
+if ( !new File(params.starTmpDir).exists()) {
+    exit 1, 'The STAR temporary folder does not exists. ('+params.starTmpDir+')\n'
 }
 if (!params.localOutDir) {
     params.localOutDir='bam'
 }
+if (!params.debug) {
+    exit 1, "Debug prefix parameter is missing."
+}
+
+
+pairFiles_ch = Channel.fromFilePairs( params.inputDir+"/"+params.inputPattern, size: 2, checkIfExists: true )
 
-pairFiles_ch = Channel.fromFilePairs( params.inputDir+"/*{1,2}.fastq.gz", size: 2, checkIfExists: true )
 
 process ERValign {
     tag "${sample}"
     
     // executor configuration
     time '8h'
-    // memory '35.GB'
+    memory '35.GB'
```

Comment:
This is not valid in nextflow. The memory should be set to '35 GB'.

See doc at https://www.nextflow.io/docs/latest/process.html#memory